.env.alfajores

```diff
@@ -44,9 +44,9 @@ AZURE_ORACLE_CENTRALUS_AZURE_KUBERNETES_RESOURCE_GROUP=baklava-oracles-centralus
 AZURE_ORACLE_CENTRALUS_KUBERNETES_CLUSTER_NAME=baklava-oracles-centralus
 AZURE_ORACLE_CENTRALUS_AZURE_REGION_NAME=centralus
 # NOTE: the oracle at index 1 is not whitelisted, so it will never be able to successfully report
-AZURE_ORACLE_CENTRALUS_ORACLE_ADDRESSES_FROM_MNEMONIC_COUNT=10
-AZURE_ORACLE_CENTRALUS_FULL_NODES_COUNT=2
-AZURE_ORACLE_CENTRALUS_FULL_NODES_DISK_SIZE=30
+AZURE_ORACLE_CENTRALUS_CELOUSD_ORACLE_ADDRESSES_FROM_MNEMONIC_COUNT=10
+AZURE_ORACLE_CENTRALUS_CELOUSD_FULL_NODES_COUNT=2
```

Comment:
iiuc this line and the one below don't need to change?

Reply:
Yes I think I was in robot mode and added them by mistake 😂